src/repr/src/row.rs

```diff
@@ -99,6 +101,42 @@ pub struct Row {
     data: SmallVec<[u8; Self::SIZE]>,
 }
 
+impl Arbitrary for Row {
+    type Parameters = ();
+    type Strategy = BoxedStrategy<Row>;
+
+    fn arbitrary_with(_: Self::Parameters) -> Self::Strategy {
```

Comment:
What if we had `type Parameters = usize` so we could specify the number of datums we want in the row? So far this only allows creating a row with a single number.

Reply:
I assume this comment will be adressed in #12125